etc/Makefile.am

```diff
@@ -40,5 +42,20 @@ pkgconfig_DATA = flux-core.pc \
 	flux-schedutil.pc
 endif
 
+noinst_SCRIPTS = \
+	completions/get_builtins.sh \
+	completions/flux.pre
+
 EXTRA_DIST = \
-	gen-cmdhelp.pl
+	gen-cmdhelp.pl \
+	$(noinst_SCRIPTS)
+
+completions/flux: $(srcdir)/completions/flux.pre
+	$(AM_V_GEN)test -d completions || mkdir completions && \
```

Comment:
I still don't think you need to `mkdir completions` when `completions/flux.pre` already exists (and therefore its parent directory) since that file is an explicit prerequisite of this rule.

Reply:
The `mkdir completions` is necessary for a vpath build since the `completions/flux.pre` isn't being installed, e.g. removing that line from the `etc/Makefile` fails a vpath build:
```bash
Making all in etc
make[1]: Entering directory '/home/garrett/flux-core/b/etc'
  GEN      completions/flux
  GEN      flux/curve
  GEN      flux/help.d/core.json
cp: cannot create regular file 'completions/flux': No such file or directory
make[1]: *** [Makefile:893: completions/flux] Error 1
```

Reply:
Yeah, sorry I missed that `completions/flux.pre` is in srcdir.